tests/testthat/test-quo.R

```diff
@@ -23,18 +23,44 @@ test_that("quo_c Test 2: `quo_c` returns error if non-quosures are input", {
   )
```

Comment:
Similar to the aforementioned commentary about the deprecation process. 

For Test 1 and Test 2 above, do we need include the usual `class = "lifecycle_warning_deprecated"` part?

Reply:
I have added a test for the deprecation warning.